source/includes/cross-version-sync.rst

```diff
@@ -1,11 +1,11 @@
-Starting in 1.7.0, ``mongosync`` can perform a cross version migration
+Starting in 1.7.0, ``mongosync`` can perform a cross-version migration
 from a lower major version source cluster to a higher major version
 destination cluster. You can migrate up to two major versions ahead. For
 example, you can synchronize a cluster running MongoDB 6.0 with a
```

Comment:
@huan-Mongo Do you think we should add a sentence like "Both clusters must still meet the minimum supported server patch releases as mentioned above." ?

Reply:
I personally don't think that's very necessary since it's not cross-version specific requirement. It applies to all the sync.